### ISSUE TITLE
Expose GLIDE number on emergency pages

### ISSUE THREAD

**Issue by JonathanGarro** [IFRCGo/go-frontend#2579]
The GLIDE steering committee is looking to improve search-ability and discoverability of sites that are using GLIDE numbers. In order to have search engines index those that use the GLIDE, like GO, the string has to be exposed within the HTML, either visibly to the end user, or just tucked away in the HTML (presumably in the meta tags). 

Depending on what others in the team think, I see the best solution as exposing it visibly within the Emergency Overview box. If that would add too much clutter, then simply within the HTML should be fine.

**Comment by tovari** [comment]
@szabozoltan69, when you are back, would you mind to check if you can add that to the html as a meta tag?

**Comment by szabozoltan69** [comment]
Not quite clear, which "meta" tag should I include the GLIDE numbers into.
Recently these "meta" tags are used:
![kép](https://user-images.githubusercontent.com/6050643/207663240-ca423e3d-7bc2-497e-b4b6-970ca0243917.png)
So the "description" meta tag is empty, shall we use that? As it is suggested [here](https://www.searchenginewatch.com/2012/05/01/how-to-use-html-meta-tags/)?

**Comment by batpad** [comment]
The way our frontend is setup, it's a bit complicated to have this information in the meta tags -- since we load the page only once and all page navigations are handled purely by the frontend. Could we just put this somewhere visibly on emergency pages? That would be the easiest - else can check with @frozenhelium what our best options are to put this data on emergency pages in a search engine friendly way.

**Comment by szabozoltan69** [comment]
Others also had this task: https://stackoverflow.com/questions/37734150/how-to-update-meta-tags-in-react-js

**Comment by batpad** [comment]
cc @frozenhelium - I think we might already use `react-helmet` - my concern is I'm not 100% sure how much Search Engines pick up changes to the meta tags that happen via Javascript - I think adding the meta tag via React Helmet (or another approach from the SO ticket) would be great, but if we want to be more sure that search engines will pick these up, I think it would be nice to have them somewhere in the `body` of the page as well.

**Comment by szabozoltan69** [comment]
What do you think about this?
![kép](https://user-images.githubusercontent.com/6050643/208641936-129e7290-abbf-4572-9494-48e9e84f59d2.png)
or (not having data):
![kép](https://user-images.githubusercontent.com/6050643/208642479-0db3511a-dd55-4095-a996-785868b00ad8.png)
@mariam-yu – is it acceptable like this? Or maybe swapping some cells?

**Comment by JonathanGarro** [comment]
hey @szabozoltan69 i think that looks great!

**Comment by szabozoltan69** [comment]
Deployed to Staging.
E.g. https://go-stage.ifrc.org/emergencies/4295
Or https://go-stage.ifrc.org/emergencies/4541

**Comment by mariam-yu** [comment]
@szabozoltan69 @JonathanGarro Shall we call it Appeal Code instead of MDR code? Also, Zoly I asked our UI designer to take a look, so if we have some tweaks, may be you can adjust them after the break? Thanks! Will share here when I have it.

**Comment by szabozoltan69** [comment]
Thanks so much, @mariam-yu for the review!
I picked the "MDR Code" idea from here:
https://github.com/IFRCGo/go-frontend/blob/develop/src/root/lang.js#L3285
Shall I change this text to "Appeal Code" (which will change the text also on DREF form), or shall I introduce a new string now?
@frozenhelium @anamariaescobar

**Comment by szabozoltan69** [comment]
It will be "Appeal Code" on both parts (here and on DREF pages also).

**Comment by szabozoltan69** [comment]
Deployed to Staging.
Check https://go-stage.ifrc.org/emergencies/4295
and https://go-stage.ifrc.org/emergencies/4541

**Comment by szabozoltan69** [comment]
Also added: Visibility – https://github.com/IFRCGo/go-frontend/pull/2596

**Comment by szabozoltan69** [comment]
![kép](https://user-images.githubusercontent.com/6050643/211606502-ea16b773-6688-4960-bfae-8bbbe3e9e5a4.png)

**Comment by szabozoltan69** [comment]
The figma plan is below. (The "visibility" request arrived later.)
![kép](https://user-images.githubusercontent.com/6050643/211885854-53e404b6-1d1d-437e-8725-8300b3f0f6b6.png)
